tests/queries/0_stateless/01474_executable_dictionary.sql

```diff
@@ -1,3 +1,11 @@
 SELECT number, dictGet('executable_complex', 'a', (number, number)) AS a, dictGet('executable_complex', 'b', (number, number)) AS b FROM numbers(1000000) WHERE number = 999999;
 SELECT number, dictGet('executable_complex_direct', 'a', (number, number)) AS a, dictGet('executable_complex_direct', 'b', (number, number)) AS b FROM numbers(1000000) WHERE number = 999999;
 SELECT number, dictGet('executable_simple', 'a', number) AS a, dictGet('executable_simple', 'b', number) AS b FROM numbers(1000000) WHERE number = 999999;
+
```

Comment:
Ok. BTW I prefer adding new tests, so when test is broken it is more easy to figure out why.

Reply:
Updated.